cmd/s3_la.go

```diff
@@ -30,9 +30,8 @@ func S3CmdLa(cmd *cobra.Command, args []string) {
 	output := string(execContainer(containerName, command))
 
 	if len(output) == 1 {
-		command := []string{"s3cmd", "ls"}
-		output := execContainer(containerName, command)
-		fmt.Printf("%s", output)
+		command = []string{"s3cmd", "ls"}
+		output = execContainer(containerName, command)
```

Comment:
`string(execContainer(containerName, command))`

Reply:
can't we just extract `output := string(execContainer(containerName, command))` and put it after the `if`?

Reply:
execContainer already return a string as per func execContainer(containerName string, cmd []string) string {